Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -76,9 +77,26 @@ public static void Init(Context activity, Bundle bundle, Assembly resourceAssemb
 			SetupInit(activity, resourceAssembly);
 		}
 
+		/// <summary>
+		/// Sets title bar visibility programmatically. Must be called after Xamarin.Forms.Forms.Init() method
+		/// </summary>
+		/// <param name="visibility">Title bar visibility enum</param>
 		public static void SetTitleBarVisibility(AndroidTitleBarVisibility visibility)
 		{
+			if((Activity)Context == null) throw new NullReferenceException("Must be called after Xamarin.Forms.Forms.Init() method");
+
 			TitleBarVisibility = visibility;
+
+			if (TitleBarVisibility == AndroidTitleBarVisibility.Never)
+			{
+				if(!((Activity)Context).Window.Attributes.Flags.HasFlag(WindowManagerFlags.Fullscreen))
+					((Activity)Context).Window.AddFlags(WindowManagerFlags.Fullscreen);
```

Comment:
@samhouts

Are you suggesting that WindowManagerFlags.Fullscreen should never be added to the window flags or not added while setting TitleBarVisibility?

What happens if the developer checks TitleBarVisibility and Window.Attributes.Flags (for whatever reason) and sees TitleBarVisibility is set to never whereas there is no fullscreen flag? It didn't make sense to me so I wanted to sync them up.